tortoise/util.go

```diff
@@ -95,23 +96,21 @@ func verifyLayer(logger log.Log, blocks []*blockInfo, getDecision func(*blockInf
 		}
 		blocks[i].validity = decision
 	}
-	if changes {
-		logger.With().Info("candidate layer is verified",
-			log.Array("blocks",
-				log.ArrayMarshalerFunc(func(encoder log.ArrayEncoder) error {
-					for i := range blocks {
-						encoder.AppendObject(log.ObjectMarshallerFunc(func(encoder log.ObjectEncoder) error {
-							encoder.AddString("decision", blocks[i].validity.String())
-							encoder.AddString("id", blocks[i].id.String())
-							encoder.AddString("weight", blocks[i].margin.String())
-							encoder.AddUint64("height", blocks[i].height)
-							encoder.AddBool("data", blocks[i].data)
-							return nil
-						}))
-					}
-					return nil
-				})),
-		)
-	}
-	return true
+	return true, changes
+}
+
+func zapBlocks(blocks []*blockInfo) zap.Field {
+	return zap.Array("blocks", log.ArrayMarshalerFunc(func(encoder log.ArrayEncoder) error {
```

Comment:
Same here. Imo we can go all the way and use `zapcore` directly instead of the aliases in the `go-spacemesh/log` package.